test/functional/test_helper.py

```diff
@@ -37,3 +39,21 @@ def is_ci() -> bool:
         `True` if current executions is on CI
     """
     return os.getenv('CI', 'false') == 'true'
+
+
+def wait_for(method: Callable, timeout: int) -> bool:
```

Comment:
I usually prefer to explicitly mention the time unit for variables (e.g. timeout -> timeout_sec). Also, it makes sense to have a default value (e.g. 5) for it and make it float, so we could provide timeouts lesser than one seconds. It also makes sense to make the interval configurable

Reply:
sure:

```
result = callable()
if result:
    return result
```

Reply:
we could even use 

```
if result := callable():
    return result
```

if we drop the support of py37 :-P

Reply:
@mykola-mokhnach 
Thanks for details.
Ah, yes, actually it popped into my head,
but as my thoughts, it could extend this method usage, but extended usage is limited for "user".

But it's ok since "user" is developer. I'll change it to suggestion. 👍